### PR TITLE
feat: default token

### PR DESCRIPTION
**Pull request by bwpikaard**
Allows you to set a token in your configuration file to authenticate for all requests without needing to set a token. Useful for testing subscriptions because you cannot provide authentication tokens in GQL Playground or Altair. Defaults to the token in the header before trying to use the config token so you can have "multiple users".

### CODE REVIEW

**Review by gankoji — DISMISSED** [review]

**Comment by bwpikaard** [comment]
Do not merge until parent branch is merged to dev.

**Review by 0c370t — APPROVED** [review]

**Pull request merged**
